docs/general/ddl/system-columns.rst

```diff
@@ -56,11 +56,15 @@ and might contain underscores in between.
 
 ``_score``
   This internal system column is available on all documents retrieved by a
-  ``SELECT`` query. It is representing the scoring ratio of the document
-  related to the used query filter and makes most sense on fulltext searches.
-
-  The scoring ratio is always related to the highest score determined by a
-  search, thus scores are not directly comparable across searches.
+  ``SELECT`` query. It represents the score of the document as returned by
+  the query filter, and makes most sense for full text queries.
```

Comment:
While being at it, I just wanted to add guidance about the "why so?" details. I hope you get the idea, and will be able to come up with something more fluent.
```suggestion
  the query filter, and makes most sense for full text queries, by using it
  as a rank indicator to know how good the expression matches the
  document, compared to others.
```